sklearn/preprocessing/discretization.py

```diff
@@ -0,0 +1,106 @@
+# Title          : discretization.py
+# Author         : Henry Lin <hlin117@gmail.com>
+# License        : BSD 3 clause
+#==============================================================================
+
+from __future__ import division
+import numpy as np
+from ..externals import six
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.utils import column_or_1d
+from sklearn.utils.validation import check_is_fitted
+
+xrange = six.moves.xrange
+
+__all__ = [
+    "FixedWidthDiscretizer"
+]
+
+
+class FixedWidthDiscretizer(BaseEstimator, TransformerMixin):
```

Comment:
I would just use `Discretizer` and later have a `strategy` option in the constructor.